mtv-plan/mtv.sw.yaml

```diff
@@ -82,12 +88,36 @@ states:
           toStateData: .planErrorMessage
         functionRef:
           refName: getPlanError
+      - name: createNotification
+        functionRef:
+          refName: createNotification
+          arguments:
+            recipients:
+              type: "entity"
+              entityRef: .recipients
+            payload:
+              title: "MTV plan is not ready for migration"
+              description: '"MTV plan " + .migrationName + " is not ready for migration. Error: " + .planErrorMessage'
+              scope: "SonataFlow"
+              topic: "MTV plan assessment workflow"
     stateDataFilter:
       output: '{assessmentResult: {passed: false, message: .planErrorMessage}}'
     end: true
   - name: PlanReady
     type: operation
-    actions: []
+    actions:
+      - name: createNotification
+        functionRef:
+          refName: createNotification
+          arguments:
+            recipients:
+              type: "entity"
+              entityRef: .recipients
+            payload:
+              title: "MTV plan is ready for migration"
+              description: '"MTV plan " + .migrationName + " is ready for migration"'
+              scope: "SonataFlow"
+              topic: "MTV plan assessment workflow"
```

Comment:
not aware of the details of the plan output, but is the plan available somewhere? If so, we might want to add a link to it (same as we do for MTA analysis output)

Reply:
see the same comment by @pkliczewski i do not have a way to link it. i do not have the console URL.